### PR TITLE
Admins can now reject shitty adminhelps

### PR DESCRIPTION
**Pull request by MrsTonedOne**
This is done as a (REJT) button on admin helps. This allows the admin to tell the user that they made a shit adminhelp. It also returns the adminhelp verb back to the user.

In order to do the latter part, I had to make a simple timers subsystem. This is designed for timed events that one may want to cancel or otherwise control later. It has a resolution of 5ds(0.5 seconds) but this can be easily changed.

image

### CODE REVIEW

**Comment by Razharas** [comment]
If anything id not return the ahelp to them instantly since if you are in rage and admins also reject you in red text more often than not next ahelp will be even more aagitated than previous

**Comment by RemieRichards** [comment]
I don't like that to return the verb you have to reject it, it may be clutter but could we have another button that simply returns the verb to them, without explicitly saying "I rejected you"

**Comment by MrStonedOne** [comment]
What would i call the button? rtn? RV?

Admins wanting to just give them back the verb is rare, majority of the time the use of this would be for "are you seeing this" or "is this valid" or "can he do that" admin helps, where the user provides no information.

The same thing goes for raz's argument, this would be used less for angry rage admin helps (we would just mute them from admin helps) and more for admin helps that provide no information.

I could remove that note at the end.

**Comment by RemieRichards** [comment]
@MrStonedOne yeah Rtn or something to that effect.
Idk the note just irked me, removing it might be better.

**Comment by optimumtact** [comment]
thank you for the timed events subsystem

**Comment by freerealestate** [comment]
This is going to add to the divisiveness between admins and players. The word "Rejected" in bold red lettering will make certain people feel insecure and resentful.

IMHO you should tweak it so you don't shit on them at all for adminhelping, and instead just skip to the part telling them the adminhelp didn't contain enough information and encourage them to be more informative so the admins can help them better.

**Comment by GunHog** [comment]
I do not see the point of this. What does this solve in a better manner than simply replying to the adminhelp? This will let the player respond to the admin immediately, and if I am correct, all active admins will see an admin PM, thus it makes no difference.

**Comment by optimumtact** [comment]
It's faster

**Comment by optimumtact** [comment]
Also this should always notify

**Comment by Stickymayhem** [comment]
The same idiots make the same dumb adminhelps with no information because they are lazy, and it slows everything down. This way it forces them to be at least somewhat helpful because they'll know they are being specifically ignored for being lazy.

**Comment by Cheridan** [comment]
adminbus kidnapped my cat and said they'll behead it if i don't merge this.

**Comment by Incoming5643** [comment]
Seems like a great way to escalate situations and add lots more drama to the metametagame. :frowning:

**Comment by optimumtact** [comment]
@Cheridan we do not negotiate with terrorists damn it

File: html/changelogs/MrStonedOne-PR-10083.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (optimumtact) [inline html/changelogs/MrStonedOne-PR-10083.yml]: God damn it MSO

**Pull request merged**